src/mangosd/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ target_link_libraries(${EXECUTABLE_NAME}
   g3dlite
   gsoap
   ${ACE_LIBRARIES}
```

Comment:
Not quite.  It's containers are still used in the database code, and it is also used I think in config file reading, and one or two other places.  We are very close once the ASIO implementation is merged, though.